internal/cloud/cloud.go

```diff
@@ -11,4 +11,6 @@ const (
 	TagRole = "constellation-role"
 	// TagUID is the tag/label key used to identify the UID of a cluster.
 	TagUID = "constellation-uid"
+	// TagInitSecretHash is the tag/label key used to identify the hash of the init secret.
+	TagInitSecretHash = "init-secret-hash"
```

Comment:
Can we prefix this with `constellation-`, or would we risk running into name length limitations?